actors/common/src/main/java/org/sunbird/learner/actors/skill/SkillmanagementActor.java

```diff
@@ -62,10 +61,65 @@ public void onReceive(Request request) throws Throwable {
         .getOperation()
         .equalsIgnoreCase(ActorOperations.GET_SKILLS_LIST.getValue())) {
       getSkillsList();
+    } else if (request.getOperation().equalsIgnoreCase(ActorOperations.UPDATE_SKILL.getValue())) {
+      updateSkill(request);
     } else {
       onReceiveUnsupportedOperation(request.getOperation());
     }
   }
+  /** Method will Update the skills from user ... */
+  private void updateSkill(Request actorMessage) {
+    ProjectLogger.log(
+        "SkillmanagementActor-updateSkill called",
+        actorMessage.getRequest(),
+        LoggerEnum.INFO.name());
+
+    String userId = (String) actorMessage.getRequest().get(JsonKey.REQUESTED_BY);
+    List<String> list = (List<String>) actorMessage.getRequest().get(JsonKey.SKILL_NAME);
+    CopyOnWriteArraySet<String> skillset = new CopyOnWriteArraySet<>(list);
```

Comment:
Can we avoid using CopyOnWriteArraySet as it is normally a costly operation?

https://docs.oracle.com/javase/7/docs/api/java/util/concurrent/CopyOnWriteArrayList.html